lib/build.rb

```diff
@@ -32,6 +32,9 @@ def initialize(token, repository, max_age)
     @potential_builds = []
     @max_age = max_age
     github_check_rate_limit(@client.last_response.headers)
+    # Keep a mapping of branch name -> PR number for PRs which are based on non-fork branches
+    # This is only used for providing the GitHub status comment, nothing else
+    @branch_to_pr_number = {}
```

Comment:
Just create an empty hash on the build class when it's constructed at the beginning of each CI pass.